src/backend/PostgresBackend.cpp

```diff
@@ -637,12 +705,91 @@ PostgresBackend::fetchLedgerDiff(
     return {};
 }
 
+NFTokenTransactions
+PostgresBackend::fetchNFTTransactions(
+    ripple::uint256 const& tokenID,
+    std::uint32_t const limit,
+    bool forward,
+    std::optional<TransactionsCursor> const& cursor,
+    boost::asio::yield_context& yield) const
+{
```

Comment:
copied and pasted from fetchAccountTransactions. Could be generalized probably, and probably without the need even for template function. but since PG is not the focus didn't want to spend time on this.